src/Pages/HomePage/Header/Header.js

```diff
@@ -61,7 +64,13 @@ function Header() {
       </div>
 
       <div className=" absolute top-7 right-12 ">
-        <Button text={t(`landing.HeaderBtnStarted`)} />
+        <Button
+          text={t(`landing.HeaderBtnStarted`)}
+          onClick={() => {
+            signInWithGoogle();
```

Comment:
you should use `.then` and after the user signs in you direct them to the home page